src/Native/Unix/System.Native/pal_io.c

```diff
@@ -37,6 +42,13 @@
 #include <sys/inotify.h>
 #endif
 
+#if defined(_AIX)
```

Comment:
Above you used `#ifdef _AIX`, here you used `#if defined(_AIX)`.  It'd be good to be consistent.  These files are already predominantly using `#ifdef` and `#ifndef`... can we use that for all of these new checks?